third-party/ios/ExecutorchLib/ExecutorchLib/runner/stats.h

```diff
@@ -12,14 +12,13 @@
 #include <cinttypes>
 #include <executorch/runtime/platform/log.h>
 #include <sstream>
-// patternlint-disable-next-line executorch-cpp-nostdinc
 #include <string>
 
 namespace executorch {
 namespace extension {
 namespace llm {
 
-struct Stats {
+struct ET_EXPERIMENTAL Stats {
   // Scaling factor for timestamps - in this case, we use ms.
   const long SCALING_FACTOR_UNITS_PER_SECOND = 1000;
```

Comment:
it is constexpr, not runtime const. Change it with `constexpr long SCALING_FACTOR_UNITS_PER_SECOND = 1000;`